packages/fast-layouts-react/src/horizontal-overflow/horizontal-overflow.utilities.ts

```diff
@@ -0,0 +1,56 @@
+import { HorizontalOverflowProps } from "./horizontal-overflow.props";
+import { ButtonDirection } from "./horizontal-overflow";
+
+export type EaseInOutQuad = (currentTime: number, startValue: number, changeInValue: number, duration: number) => number;
+export type AnimateScroll = () => void;
+
+export function getItems(props: HorizontalOverflowProps): React.ReactNode[] {
+    const children: React.ReactNode[] = Array.isArray(props.children)
+        ? props.children
+        : props.children
+        ? [props.children]
+        : [];
+
+    return children.filter((child: JSX.Element) => {
+        return child.props && child.props.slot !== ButtonDirection.previous && child.props.slot !== ButtonDirection.next;
+    });
+}
+
+/**
+ * Inspired by the github gist contribution: https://gist.github.com/andjosh/6764939
+ */
+export function scrollLeft(element: HTMLDivElement, left: number, duration: number): void {
+    const start: number = element.scrollLeft;
+    const change: number = left - start;
+    const startDate: number = +new Date();
```

Comment:
is that `+` supposed to be in the `new Date()` expression?

Reply:
I see what this does - I'd recommend revising to something more human readable: https://stackoverflow.com/questions/14255664/new-date-is-this-good-practice